### ISSUE TITLE
Slug auto-generation broken (wagtail 5.0.1)

### ISSUE THREAD

**Issue by EmCeeEs** [infoportugal/wagtail-modeltranslation#390]
### Description
After adding _title [en]_ for a new page in admin interface -> Save fails, due to _slug [en]_ not being auto-generated:
```
The page could not be saved due to validation errors
```

I noticed that on focusing the _title [en]_ form field, the following error is logged to browser console:
```
wagtail_translated_slugs.js:28 Uncaught ReferenceError: cleanForSlug is not defined
    at HTMLInputElement.<anonymous> (wagtail_translated_slugs.js:28:30)
    at HTMLInputElement.dispatch (jquery-3.6.0.min.js?v=cf233667:2:43003)
    at v.handle (jquery-3.6.0.min.js?v=cf233667:2:40998)
```

### Env
python 3.10.6
django 4.2.3
wagtail 5.0.1
wagtail-modeltranslation 0.14.0

**Comment by EmCeeEs** [comment]
Jup, `cleanForSlug`  has been [moved](https://github.com/wagtail/wagtail/commit/510a6bf350e4124193e63b076398010a94e019d8).
As a test: Monkey patching it in the browser console fixes the issue.

**Comment by DiogoMarques29** [comment]
Temporary fix in #393
